### DISCUSSION TITLE
Rendering symbols to HTML codes?

### DISCUSSION THREAD

**Discussion by blyxyas** [sunng87/handlebars-rust#564] [Q&A] [answered]
Hi, I have this example data:

```
json!({
    "my_data": "It uses symbols <3"
})
```

And after using `reg.render`, it converts that into `It uses symbols &gt;3`, how can I deactivate this feature?

**Comment by sunng87** [comment, marked as answer]
Yes, by default handlebars escapes html entity values. To disable this behaviour, call `register_escape_fn(no_escape)` on handlebars registry.
